react/features/invite/components/info-dialog/web/PasswordForm.js

```diff
@@ -162,7 +172,15 @@ class PasswordForm extends Component<Props, State> {
      * @returns {void}
      */
     _onEnteredPasswordChange(event) {
-        this.setState({ enteredPassword: event.target.value });
+        const enteredPassword = event.target.value;
+
+        // if digits are enabled we want to input only digits and
+        // allow empty string so user can delete its input
+        if (!this.props.passwordNumberOfDigits
+            || ((enteredPassword.length === 0 || /^\d+$/.test(enteredPassword))
+                && enteredPassword.length <= this.props.passwordNumberOfDigits)) {
```

Comment:
How about using maxLength on <input /> then now that there’s a separate configuration value for it? Actually I think inputs have a validation attribute you could use instead. Mobile might be the same.

Reply:
What validation attribute?

Reply:
I looked it up and it’s pattern. Maybe that’ll work.

Reply:
I just pushed that change.
Here how it looks like:
![image](https://user-images.githubusercontent.com/3263098/58102110-af8e4600-7bd8-11e9-85b2-75993c9c0193.png)

Reply:
I guess you don't want that. Looked up the solution and you'd have to add an onInvalid handler, so maybe the previous changeset is what you prefer.

Reply:
I'm lost. I'll leave it that way, its a little bit different than mobile, but whatever ... I got a blessing from design and for this one ...

Reply:
Wait, so does it not enforce? What happens if you submit? Does it still submit with the letters?

Reply:
No, you see a screenshot when trying to submit.